### DISCUSSION TITLE
For an unknown reason

### DISCUSSION THREAD

**Discussion by rijyaaru** [game-difficulty/2048EndgameTablebase#1] [General]
I encounter an error when running main.py:

Process finished with exit code -1073740791 (0xC0000409)
Additionally, I do not see any executable (EXE) files in the directory. I hope to get some guidance from you on this matter.

If you need any further assistance, feel free to ask!

**Comment by game-difficulty** [comment]
The exe file is in the release. Can you provide more detailed information about the program error exit? For example, what operations did you perform, what stage of program execution did you get to, and what system are you on?
